ga4gh/datamodel/variants.py

```diff
@@ -423,6 +440,11 @@ def getVariants(self, referenceName, startPosition, endPosition,
         # protocol version 0.6
         if callSetIds is None:
             callSetIds = []
+        else:
+            for call in callSetIds:
+                if call not in self._callSetIds:
+                    raise exceptions.CallSetNotInVariantSetException(
```

Comment:
This exception is defined. but i don't think it was actually ever used. it is currently in ga4gh/exception.py
```python
class CallSetNotInVariantSetException(NotFoundException):
    """
    Indicates a request was made for a callSet not in the actual variantSet
    """
    def __init__(self, callSetId, variantSetId):
        self.callSetId = callSetId
        self.variantSetId = variantSetId

    def getMessage(self):
        message = "callSet '{0}' not in variantSet '{1}'".format(
            self.callSetId, self.variantSetId)
        return message
```
i m inclined not to make changes in this PR, instead raise a minor separate issue, fix it after PR #294 is merged